### ISSUE TITLE
Setting the name of a remote to an URL prevents the creation of the remote

### ISSUE THREAD

**Issue by codeZeilen** [hpi-swa/Squot#161]
The debugger shows an exception when trying to create a directory with the name of the remote, which fails as the name is an URL.

**Comment by j4yk** [comment]
Just so I understand this correctly: Did you accidentally paste the URL into the prompt for the remote name? And you would like this to be catched in a helpful way?

**Comment by codeZeilen** [comment]
Yes, I accidentially pasted it in there (you know first clicking then reading...). And yes I think it would be helpful to try to prevent this little mistake in some way :)

**Comment by j4yk** [comment]
@codeZeilen Can you please have a look at the changes I have made? Does it help to a) enter the URL first and b) enter both the URL and the name in the same dialog?

**Comment by codeZeilen** [comment]
This looks like a very good idea, it also illustrates the relation between origin name and its url.
